packages/core-react/src/components/Button/index.js

```diff
@@ -45,7 +45,7 @@ Button.propTypes = {
 
 Button.defaultProps = {
   Tag: 'button',
-  type: 'primary',
+  kind: 'primary',
```

Comment:
I understand how this conflicts with HTML but I'm not digging the `kind` name. Can we do variant? Also, this would be a breaking change right? It looks like 4 other teams/repos are importing ray (but i guess they may/may not be using the React implementation of it). Otherwise, I'm OK with this update, but I'd want to hear from other folks about the name of this prop. (@jamesdamild would love your feedback as well :-P)

Reply:
`variant` works as well. not issues there. just can't be `type`

Reply:
+1 on variant

fwiw material ui uses `variant` and semantic ui react has individual boolean props for each of the variants